lib/init/rc.shutdown

```diff
@@ -8,10 +8,8 @@ main() {
     PATH=/sbin:/bin:/usr/sbin:/usr/bin
 
     log "Waiting for services to stop..."; {
-        if command -v sv && [ "$(pgrep runsvdir)" ]; then
-            sv -w196 force-stop /var/service/*
-            sv exit /var/service/*
-        fi
+        sv -w196 force-stop /var/service/* >/dev/null 2>&1
+        sv exit /var/service/* >/dev/null 2>&1
     }
```

Comment:
Even simpler.

`} >/dev/null 2>&1`

Basically, make it affect the whole block at once!

Reply:
little by little I will get there